fw.lua

```diff
@@ -296,6 +300,16 @@ local function _log_event(self, request_client, request_uri, rule, match)
 		t.rule.var = rule.var
 	end
 
+	if self._event_log_save_post_data then
+		t.post_body = collections["REQUEST_BODY"]
+	end
+
+	if (table.getn(self._event_log_ngx_vars) ~= 0) then
+		for _, k in ipairs(self._event_log_ngx_vars) do
+			t[k] = ngx.var[k]
```

Comment:
Might want to put these in a sub-table so we don't inadvertantly stomp on an already-set key.